### PR TITLE
Pulsating animation of circle

### PR DESCRIPTION
**Pull request by bhoomika08**

### CODE REVIEW

File: Animations/App.js [hunk 1]
```diff
@@ -1,106 +1,25 @@
...
+    </View>
   );
 };
 
-const styles = StyleSheet.create({
-  scrollView: {
-    backgroundColor: Colors.lighter,
+const styles = {
+  mainContainer: {
+    marginTop: isIOSPlatform ? getStatusBarHeight() : 0,
```
Comment (gash48) [inline Animations/App.js:26]: Need ?

File: Animations/package.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
   },
   "dependencies": {
     "react": "16.13.1",
-    "react-native": "0.63.4"
+    "react-native": "0.63.4",
+    "react-native-status-bar-height": "^2.6.0"
```
Comment (gash48) [inline Animations/package.json:15]: Not required

File: Animations/src/components/pulsating-animation.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  const fadeAnim = new Animated.Value(minValue);
+
+  useEffect(() => {
+    Animated.loop(
+      Animated.sequence([
+        Animated.timing(fadeAnim, {
+          toValue: maxValue,
+          duration: animDuration,
+          easing: Easing.circle,
+          useNativeDriver: false,
```
Comment (gash48) [inline Animations/src/components/pulsating-animation.js:24]: Always set to true for better performance

File: Animations/src/components/pulsating-animation.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        }),
+        Animated.timing(fadeAnim, {
+          toValue: minValue,
+          duration: animDuration,
+          easing: Easing.circle,
+          useNativeDriver: false,
```
Comment (gash48) [inline Animations/src/components/pulsating-animation.js:30]: Same

File: Animations/src/components/pulsating-animation.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        }),
+      ]),
+    ).start();
+  }, [fadeAnim]);
+
+  const BackgroundColorConfig = fadeAnim.interpolate({
+    inputRange: [minValue, maxValue],
+    outputRange: [blue, pink],
```
Comment (gash48) [inline Animations/src/components/pulsating-animation.js:38]: Nice color choice

File: Animations/src/components/pulsating-animation.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  });
+
+  const animatedStyle = {
+    ...styles.circleShapeView,
+    opacity: fadeAnim,
+    backgroundColor: BackgroundColorConfig,
+  };
+
+  const {container, headerContainer, heading} = styles;
```
Comment (gash48) [inline Animations/src/components/pulsating-animation.js]: Destructure styles at the top

**Review by gash48 — CHANGES_REQUESTED** [review]
Requested Changes

File: Animations/App.js (hunk 1 above)
Comment (bhoomika08) [reply Animations/App.js:26]: Added a heading for each animation. That's why added margin top for IOSPlatform.

File: Animations/src/components/pulsating-animation.js (hunk 3 above)
Comment (bhoomika08) [reply Animations/src/components/pulsating-animation.js:24]: On setting useNativeDriver: true, following error occurred.
`Error: Style property 'backgroundColor' is not supported by native animated module.`

File: Animations/src/components/pulsating-animation.js (hunk 4 above)
Comment (bhoomika08) [reply Animations/src/components/pulsating-animation.js:30]: On setting useNativeDriver: true, following error occurred.
`Error: Style property 'backgroundColor' is not supported by native animated module.`

File: Animations/src/components/pulsating-animation.js (hunk 6 above)
Comment (bhoomika08) [reply Animations/src/components/pulsating-animation.js]: Done

File: Animations/App.js (hunk 1 above)
Comment (bhoomika08) [reply Animations/App.js:26]: Will get rectify in navigation router.

File: Animations/package.json (hunk 2 above)
Comment (bhoomika08) [reply Animations/package.json:15]: Will get rectify in navigation router.

File: Animations/src/components/pulsating-animation.js (hunk 3 above)
Comment (gash48) [reply Animations/src/components/pulsating-animation.js:24]: Due to JS only property modification

**Pull request merged**